classes/local/importvideosmanager.php

```diff
@@ -419,4 +456,137 @@ public static function duplicate_videos($sourcecourseid, $targetcourseid, $cours
         // Finally, inform the caller that the tasks have been created.
         return true;
     }
+
+    /**
+     * Helperfunction to get the series object and process it with rendered later on in the summary.
+     *
+     * @param int $sourcecourseid The source course id.
+     *
+     * @return object
+     */
+    public static function get_import_source_course_series_object($sourcecourseid) {
```

Comment:
What is the use case of this function?
It seems as it does exactly like `get_course_series` from `apibridge`. Please try to use existing code as much as possible.

Reply:
In order to maintain consistency in `classes/local/importvideos_step2_form_acl.php:62`, I decided to use the function `importvideosmanager::get_import_source_course_series_object` in which I get the apibridge instance and with new changes I only call the `get_course_series` method only.
As you can see there is no apibridge instance inside `importvideos_step2_form_acl` or any other importvideos steps. However, if you see no point of doing that we can change it and bring it in.

Reply:
I'd prefer to change it as it makes the code more complicated. It's totally fine to use the apibridge in the form even if its not used during the other import steps.